src/components/bcros/filing/item/ChangeOfAddress.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <BcrosFilingCommonTemplate :filing="filing" data-cy="change-of-address">
```

Comment:
This component seems empty if isFutureEffective is false, is it easier to just not render it when thats the case or lift the if statement higher in this component? or is there some content coming that changes this